### ISSUE TITLE
Update librdkafka?

### ISSUE THREAD

**Issue by chuck-alt-delete** [SeisoLLC/zeek-kafka#32]
The version of librdkafka seems fixed at 1.4.2, which is about a year and a half old at this point. Any reason not to use a newer version? Maybe this can be incorporated into CI?

Also, all the examples show compiling librdkafka from source. Any reason not to install it with a package manager?

**Comment by JonZeolla** [comment]
@chuck-alt-delete yeah that shouldn't be an issue.  To run the tests with a new version the only update is to https://github.com/SeisoLLC/zeek-kafka/blob/main/docker/docker-compose.yml#L49

**Comment by chuck-alt-delete** [comment]
Thanks Jon!

Sort of related is:
- https://github.com/SeisoLLC/zeek-kafka/issues/33

If docker image artifacts are published somewhere, maybe it makes sense to have images with different versions of zeek and different versions of librdkafka so people can pull off the shelf what version they need?
